exporter/clickhouseexporter/exporter_logs_test.go

```diff
@@ -121,6 +121,7 @@ func TestExporter_pushLogsData(t *testing.T) {
 	})
 }
 
+// nolint:unparam // not need to check this func
```

Comment:
Why is this needed? All the parameters seem to be used...

Reply:
make lint:
![image](https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/35491170/8e521d58-7b8a-4f01-8ddd-b97a32e1b7e2)

Reply:
Ok, so why do you need `fns` argument if it's never passed?